unit1/lesson12.rst

```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
+Objectives
+----------
+
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
+- Define HTML attributes and how they are used
+- Describe how whitespace in HTML documents effects rendering
+- Describe inline and block display; for common elements identify their default display
+- Describe HTML document structure and associated elements: doctype, <html>, <head>, <body>, <title>
+- Write a properly-structured document head.
+- Describe the purpose of, and use additional HTML tags: <ul>, <ol>, <li>, <table>
+- Write HTML comments
+
+Git
+^^^
+
+- Describe the main functionality provided by a version control system (VCS)
+- Describe the practical applications of this functionality: revision history, collaboration, backup
+- Explain what a repository is
+- Explain what a commit is
+- Describe the basic commit flow and commands: status, add, commit, log
+- Initialize a new Git repository
+- Commit to a local repository
+- Clone a project from a remote Git repository
```

Comment:
Students should also know how to accomplish the reverse of this - how to create a remote repo (or branch) from a local repo.  (Not sure if this topic belongs here or in the next lesson).

Reply:
It seems that this is included in the last objective for class 13: "Create and push to a remote repository." The only possible addition to that one to make it fully clear could be something about how to connect a remote repo to a local repo, which is implicit in this objective but could be explicitly stated.

Reply:
I can reword that objective in Lesson 13 to make it clearer that that is what it means.